procedures/foreman/release.md.erb

```diff
@@ -49,13 +51,15 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 # Packaging a release
 
 [Background documentation](https://projects.theforeman.org/projects/foreman/wiki/Release_Process#Packaging-a-release)
-
+* [ ] Make sure all nightly builds of packages are removed from the releases tags
 * [ ] Update [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) and [deb/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_deb_packaging) branches
 * [ ] Trigger [release_packages in Jenkins](https://ci.theforeman.org/job/release_packages/) by calling [release_packages](https://github.com/theforeman/theforeman-rel-eng/blob/master/release_packages)
 * [ ] Tag `foreman-release` into `foreman-client-<%= short_version %>-rhel7`, as this doesn't happen automatically for that one tag: `koji tag-build foreman-client-<%= short_version %>-rhel7 foreman-release-<%= version %>-1.el7`<% if is_rc %> (make sure the build matches the release)<% end %>
+* [ ] Make sure the [Nightly plugins pipeline](https://ci.theforeman.org/job/foreman-plugins-nightly-rpm-pipeline) has passed recently
```

Comment:
Normally nightly is not important for a release

Reply:
I think this is important for branching process so we branch a working codebase.

Reply:
Correct, but that's `branch.md.erb`, not `release.md.erb`.

Reply:
As I recall, there was something that failed for me if this hadn't passed recently... Lemme see if I can find the part.

Reply:
Hrm, not sure what was blocking me but wasn't this. Will remove and if I recall what it is I'll reupdate.